form-builder/index.html

```diff
@@ -0,0 +1,956 @@
+                                <input type="checkbox" class="form-check-input"
+                                    data-bind="attr: { id: 'reviewPage_' + sectionId() }, checked: reviewPage">
+                            </div>
+                            <div style="display: inline-block; margin-left: 1rem" class="form-group form-check">
+                                <label class="form-check-label" data-bind="attr: { for: 'newPage_' + sectionId() }">New
+                                    Page?</label>
+                                <input type="checkbox" class="form-check-input"
+                                    data-bind="attr: { id: 'newPage_' + sectionId() }, checked: newPage">
+                            </div>
+                            <div class="question-list" data-bind="foreach: questions">
+                                <div class="card mb-3 question"
+                                    data-bind="attr: { id: 'question_' + questionId() }, css: { 'question-odd': $index() % 2 === 0, 'question-even': $index() % 2 !== 0 }">
+                                    <div class="card-header d-flex justify-content-between align-items-center">
+                                        <span class="drag-handle">&#x2630;</span>
+                                        <h4 data-bind="text: 'Question: ' + questionId()"></h4>
+                                        <div>
+                                            <span type="button" class="duplicate-button"
+                                                data-bind="click: $parent.duplicateQuestion.bind($parent)">
+                                                <i class="fa-solid fa-copy fa-2x"></i>
+                                            </span>
+                                            <button type="button" class="remove-button"
+                                                data-bind="click: $parent.removeQuestion.bind($parent)">&times;</button>
+                                        </div>
+                                    </div>
+                                    <div class="card-body">
+                                        <div class="form-group">
+                                            <label class="required"
+                                                data-bind="attr: { for: 'questionId_' + questionId() }">Question
+                                                ID</label>
+                                            <input type="text" class="form-control"
+                                                data-bind="attr: { id: 'questionId_' + questionId() }, value: questionId"
+                                                required>
+                                        </div>
+                                        <div class="form-group">
+                                            <label class="required"
+                                                data-bind="attr: { for: 'questionTitle_' + questionId() }">Question
```

Comment:
The same thing for question `Title`, add red Asterisk indicate as required field on Title field